dolo/compiler/model_numeric.py

```diff
@@ -179,6 +179,14 @@ def _repr_html_(self):
             equations = {"dynare": self.symbolic.equations}
         else:
             equations = self.symbolic.equations
+            # Create definitions equations and append to equations dictionary
+            definitions = self.symbolic.definitions
+            tmp = []
+            for deftype in definitions:
+                tmp.append(deftype + '=' + definitions[deftype])
+            definitions = {'definitions': tmp}
+            equations.update(definitions)
```

Comment:
Ahh sorry, for some reason I thought this appended the definitions to `self.symbolic.equations` -- we don't want that.

If this is all happening with a local copy of `self.symbolic.equations` then it's all good!